purchase_work_acceptance/models/account_invoice.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    require_wa = fields.Boolean(
+        default=lambda self: self._get_require_wa(),
```

Comment:
Can this be compute field (store=False), so we don't need to create database field?